src/main/scala/mesosphere/marathon/MarathonConf.scala

```diff
@@ -137,6 +137,11 @@ trait MarathonConf
     case None => Set(ResourceRole.Unreserved)
   }
 
+  lazy val defaultEnforceGroupRole = opt[String](
+    name = "default_enforce_group_role",
+    descr = "The default for `enforceRole` in top level groups.",
+    default = Some("false"))
```

Comment:
Huh, `false`? Not `Off`? First spec casualty? :D

Reply:
Yeah. So I don't know what's right here.